src/pages/resultsView/enrichments/MiniScatterChart.tsx

```diff
@@ -72,8 +87,16 @@ export default class MiniScatterChart extends React.Component<IMiniScatterChartP
         return (
             <div className="posRelative">
                 <div className="borderedChart inlineBlock">
-                    <VictoryChart containerComponent={<VictorySelectionContainer responsive={false}
-                        onSelection={(points: any, bounds: any, props: any) => this.handleSelection(points, bounds, props)} 
+                    <div className="btn-group" style={{position:'absolute', zIndex:10, right: 26 }} role="group">
```

Comment:
@adamabeshouse  has a component for this now I think.

Reply:
Updated